pages/docs/tutorials/getting-started/request-reply.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Request/reply pattern
+weight: 40
+---
+
+In this tutorial, you'll learn how to implement the request/reply pattern in a AsyncAPI document using a simple pong-pong example.
+
+Before we begin, it would be beneficial and easy to follow if you have a basic understanding of AsyncAPI and event-driven architecures. If you don't or need a refresher, you can refer to the [Event-Driven architecture](/docs/tutorials/getting-started/event-driven-architectures) document.
```

Comment:
```suggestion
Before we begin, it would be beneficial for you to have a basic understanding of AsyncAPI and Event-Driven Architectures (EDA). If you need a refresher, refer to our [Event-Driven Architecture](/docs/tutorials/getting-started/event-driven-architectures) document.
```